contrib/broker/k8s/Dockerfile

```diff
@@ -20,13 +20,16 @@ RUN export DEBIAN_FRONTEND=noninteractive \
  && apt-get update \
  && apt-get install -y \
     ca-certificates \
+    curl \
+ && curl -Ls -o /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+ && mkdir -p /opt/services \
+ && tar -C /opt/services -x -z -f /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    --strip-components=1 linux-amd64/helm \
+ # cleanup
```

Comment:
I think he means the comment - yes it will terminate the RUN - this was recently "fixed" in Docker to act this way.  We should remove the comment.